src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -1842,6 +1844,7 @@ namespace OutputReportPredefined {
         pdchDXCoolCoilCOP = newPreDefColumn(pdstDXCoolCoil, "Standard Rated Net COP [W/W]");
         pdchDXCoolCoilEERIP = newPreDefColumn(pdstDXCoolCoil, "EER [Btu/W-h]");
         pdchDXCoolCoilSEERIP = newPreDefColumn(pdstDXCoolCoil, "SEER [Btu/W-h]");
+        pdchDXCoolCoilSEERDefIP = newPreDefColumn(pdstDXCoolCoil, "SEER Default [Btu/W-h]");
```

Comment:
How about calling this "SEER User" and "SEER Default" (or SEER Standard") to make the difference very clear?

Reply:
Done.